app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -245,19 +260,19 @@ class ConversationListViewModel @Inject constructor(
 
                 MutedConversationStatus.OnlyMentionsAndRepliesAllowed ->
                     when (it) {
-                        is Group -> it.unreadEventCount.containsKey(UnreadEventType.MENTION)
-                                || it.unreadEventCount.containsKey(UnreadEventType.REPLY)
+                        is Group -> it.unreadEventCount.containsKey(UnreadEventType.MENTION) ||
+                            it.unreadEventCount.containsKey(UnreadEventType.REPLY)
 
-                        is OneOne -> it.unreadEventCount.containsKey(UnreadEventType.MENTION)
-                                || it.unreadEventCount.containsKey(UnreadEventType.REPLY)
+                        is OneOne -> it.unreadEventCount.containsKey(UnreadEventType.MENTION) ||
+                            it.unreadEventCount.containsKey(UnreadEventType.REPLY)
 
                         else -> false
                     }
 
                 else -> false
-            }
-                    || (it is Connection && it.connection.status == ConnectionState.PENDING)
-                    || (it is Group && it.hasOngoingCall)
+            } ||
+                (it is Connection && it.connection.status == ConnectionState.PENDING) ||
+                (it is Group && it.hasOngoingCall)
```

Comment:
I know that thius is out of the scope of this PR but when I am reading this I had to thing for a sec why we are doing this checks for the keys, I know that this is due to the fact that we do not want to take this stuf in counter when looking for unread conversations, but maybe if you got the time it would be nice to fix it with some meaningfull local variable explaining it abit more

Reply:
I think was just auto-formatting that moved some lines 🤔 , no logic changed, so I can't tell why we are doing here